pytest_mypy_plugins/tests/test_configs/root_pyproject2.toml

```diff
@@ -0,0 +1 @@
+# This file has no `[tool.pytest-mypy-plugins.mypy-config]` existing config
```

Comment:
We can just update the comment in https://github.com/typeddjango/pytest-mypy-plugins/blob/master/pytest_mypy_plugins/tests/test_configs/pyproject2.toml

No need to create two empty files.